git_history_of_project/git_history_of_project.md

```diff
@@ -0,0 +1,620 @@
+<!--
+
+author:  Elizabeth Drellich
+email:    drelliche@chop.edu
+version:  1.0.0
+language: en
+narrator: UK English Female
+title: Exploring the History of your Git Repository
+comment:  This module will teach you how to look at past versions of your work on Git, compare versions, and return to former versions.
+long_description: You know that version control is important. You know how to save your work to your Git repository. Now you are ready to look at and compare different versions of your work. In this module you will you will learn how to navigate through the commits you have made to Git. You will also learn how to compare current code with past code, and, if necessary, revert to an earlier version of your work.
+
+@estimated_time: 50 minutes
+
+@learning_objectives  
+After completion of this module, learners will be able to:
+
+- Identify and use the HEAD of a repository.
+- Identify and use Git commit numbers.
+- Compare versions of tracked files.
+- Restore old versions of files.
+
+@end
+
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+
+script: https://kit.fontawesome.com/83b2343bd4.js
+
+-->
+
+# Exploring the History of your Git Repository
+
+
+<div class = "overview">
+
+## Overview
+@comment
+
+**Is this module right for me?** @long_description
+
+**Estimated time to completion:**
+@estimated_time
+
+**Pre-requisites**
+
+This module is for you if:
+
+* You know know how to create a Git repository and make changes using `add` and `commit` from a command line interface (CLI)
+* You want to learn how to compare different versions of your work and restore files to previous versions.
+
+
+**Learning Objectives**
+
+@learning_objectives
+
+</div>
+
+## Lesson Preparation
+
+The purpose of this module is to learn how to explore the history of a Git repository that has already been created. We are going to download Dracula's repository in which he started documenting the pros and cons of moving with his friends to Mars. Open up your command line interface and type:
```

Comment:
```suggestion
The purpose of this module is to learn how to explore the history of a Git repository that has already been created. We are going to download Dracula's repository in which he started documenting the pros and cons of moving with his friends to Mars. Open up your command line interface, navigate to the directory you'd like to clone a repository into, and type:
```

Also at some point around here, you should copy/paste the "but if you're in windows..." caveat from git create/track